pkg/scheduler/schedule_one_test.go

```diff
@@ -2514,6 +2529,70 @@ func Test_prioritizeNodes(t *testing.T) {
 				},
 			},
 		},
+		{
+			name:  "plugin with skip in preScore shouldn't continue to score pod",
+			pod:   &v1.Pod{},
+			nodes: []*v1.Node{makeNode("node1", 1000, schedutil.DefaultMemoryRequest*10), makeNode("node2", 1000, schedutil.DefaultMemoryRequest*10)},
+			pluginRegistrations: []st.RegisterPluginFunc{
+				st.RegisterQueueSortPlugin(queuesort.Name, queuesort.New),
+				st.RegisterScorePlugin(noderesources.BalancedAllocationName, frameworkruntime.FactoryAdapter(feature.Features{}, noderesources.NewBalancedAllocation), 1),
+				st.RegisterScorePlugin("Node2Prioritizer", st.NewNode2PrioritizerPlugin(), 1),
+				st.RegisterBindPlugin(defaultbinder.Name, defaultbinder.New),
+				st.RegisterPluginAsExtensions("FakePreScoreAndScorePlugin", st.NewFakePreScoreAndScorePlugin("FakePreScoreAndScorePlugin",
+					&st.FakePreScorePlugin{Status: framework.NewStatus(framework.Skip, "fake skip")},
+					&st.FakeScorePlugin{FakeScore: 100},
```

Comment:
```suggestion
					&st.FakeScorePlugin{Status: framework.NewStatus(framework.Error, "this score function shouldn't be executed because this plugin returned Skip in the PreScore.")},
```

let's make this plugin return error in the score so that other people can easily understand why this test case is failing in the future.

Reply:
Do same for the another test case, please.